Dockerfile.arm

```diff
@@ -1,11 +1,9 @@
 FROM plugins/base:multiarch
-MAINTAINER Drone.IO Community <drone-dev@googlegroups.com>
 
-LABEL org.label-schema.version=latest
-LABEL org.label-schema.vcs-url="https://github.com/drone-plugins/drone-downstream.git"
-LABEL org.label-schema.name="Drone Downstream"
-LABEL org.label-schema.vendor="Drone.IO Community"
-LABEL org.label-schema.schema-version="1.0"
+LABEL maintainer="Drone.IO Community <drone-dev@googlegroups.com>" \
```

Comment:
Could this be moved to the base image?

Reply:
AFAIK it's best practice to define the maintainer on every image, but possibly it can also be just added on the base image.